locales/en/filters.json

```diff
@@ -63,6 +63,10 @@
 		"filter.16.name": "AdGuard French filter",
 		"filter.16.description": "Liste FR + AdGuard French filter. Filter list that specifically removes ads on websites in the French language."
 	},
+	{
+		"filter.17.name": "AdGuard URL Tracking filter",
```

Comment:
Guys, prior to merging this, we need to upload this to Crowdin AND supply translations to German, Russian, Japanese, Chinese, Korean, French.